packages/twenty-server/src/engine/core-modules/file/guards/file-path-guard.ts

```diff
@@ -15,20 +16,31 @@ export class FilePathGuard implements CanActivate {
   async canActivate(context: ExecutionContext): Promise<boolean> {
     const request = context.switchToHttp().getRequest();
     const query = request.query;
-
+    
     if (!query || !query['token']) {
       return false;
     }
-
-    const payload = await this.jwtWrapperService.verifyWorkspaceToken(
+    let payload:any ={}
+    try{
+     payload = await this.jwtWrapperService.verifyWorkspaceToken(
       query['token'],
       'FILE',
-    );
-
+    )
+  }catch (error) {
+    if(error instanceof AuthException && error.code === AuthExceptionCode.UNAUTHENTICATED ){
+      return false
+    }
+    else{
```

Comment:
no need for else as you are already returning

if (error instance of...) {
return
}

throw ...

Reply:
actually forget about this comment as I think we should have another strategy :)